### PR TITLE
[add] V1 namespace

### PR DESCRIPTION
**Pull request by connorlay**
API is now namespaced into V1.

### CODE REVIEW

File: app/controllers/api/v1/application_controller.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    private
+
+    def current_user
+      @current_user ||= User.authenticate(access_token)
+    end
+    helper_method :current_user
+
+    def authenticated?
+      !current_user.nil?
```
Comment (Fosome) [inline app/controllers/api/v1/application_controller.rb]: `current_user.present?`

Comment (connorlay) [inline app/controllers/api/v1/application_controller.rb]: I always forget about `present?` :sweat_smile:

**Comment by Fosome** [comment]
:+1:

**Comment by mackermedia** [comment]
:+1:

**Pull request merged**